packages/shared/tailwind/colors.js

```diff
@@ -180,6 +182,43 @@ const colors = {
     70: '#2E4886',
     80: '#2E4886',
     90: '#253B76',
+    darkLabel: '#FFFFFF',
+  },
+  reddit: {
```

Comment:
With this I think we need to modify the `tailwind.config` variants of reddit/linkedin etc?
Although maybe that was more preferred as they only have 1 color really?
So you don't really need the color schema right.

Reply:
True, these colors are only used for buttons and we don't really need the color schema. We had kind of a dichotomy here, where twitter for example only defined 1 color and whatsapp and facebook more of a color scheme.

I think keeping the values here makes more sense than in the `tailwind.config.js`, so I'm going to remove the themes defined there in favor of the colors defined in this file.

Reply:
Ah I think Twitter had multiple when it was blue probably just something missed/choose the easy way when doing the black

Reply:
I meant more should we not rather go for single entries in `tailwind.config` vs having the same value here everywhere.
Unless we use the specific implementation anywhere?